chapters/dashboard/log-stream.md

```diff
@@ -0,0 +1,29 @@
+## LogStream
+
+##### Jina’s LogStream lets you monitor and debug your flows easily. You can audit your running flow live and  filter, group and search based on component (Pod) or log level / message.
+
+![logging](logging.gif)
+
+### Table View:
+
+In Table View you can check out all your logs in chronological order
+
+![table-view.png](table-view.png)
+
+### Group by Pod:
+
+You can also group your logs by the component that produced them
+
+![group-by-pod.png](group-by-pod.png)
```

Comment:
you could add an outline on top of the image to highlight the functionality. eg. around the group by pod or group by level dropdown